util/Migrator/DbScripts/2024-10-31-00_DeviceActivation.sql

```diff
@@ -0,0 +1,112 @@
+-- add column
```

Comment:
Would like to add this at top of script to avoid being deadlock victim
SET DEADLOCK_PRIORITY HIGH;